apps/web/pages/booking/[uid].tsx

```diff
@@ -1121,7 +1004,7 @@ export async function getServerSideProps(context: GetServerSidePropsContext) {
     customInputs: customInputSchema.array().parse(eventTypeRaw.customInputs),
   };
 
-  if (eventType.metadata?.disableSuccessPage && eventType.successRedirectUrl && !cancel) {
+  if (eventType.successRedirectUrl && isSuccessBookingPage && !cancel) {
```

Comment:
This fixes a bug where if "Disable success Page" was enabled, the user can't see the booking details from bookings/upcoming. The page always redirects to custom success page.